scripts/azure-pipelines/azure-pipelines.yml

```
@@ -5,7 +5,20 @@ variables:
   windows-pool: 'PrWin-2020-06-30'
   linux-pool: 'PrLin-2020-06-30'
 
-jobs:
+stages:
+- stage: check_formatting
+  pool: $(windows-pool)
+  jobs:

```

```suggestion
  pool: $(windows-pool)
  displayName: 'Check Formatting'
  jobs:

```

Didn't say C++ here because if/when we add manifest formatting it'll go here too (left the other displayName alone)

---

yeah I'm actually adding manifest formatting right now since there's no reason not to.